java_gen/templates/of_interface.java

```diff
@@ -37,7 +37,7 @@
 //:: include("_imports.java", msg=msg)
 
 public interface ${msg.name}${ "<%s>" % msg.type_annotation if msg.type_annotation else ""} extends ${", ".join(msg.all_parent_interfaces)} {
-//:: for prop in msg.members:
+//:: for prop in sorted(msg.members):
```

Comment:
changing the dictionary comprehension in java_model.py#L399 may also take care of the changes here?

Reply:
yes!